analytic_engine/src/meta/details.rs

```diff
@@ -107,20 +152,20 @@ define_result!(Error);
 
 #[async_trait]
 trait MetaUpdateLogEntryIterator {
```

Comment:
It seems we wanted to rename it to `MetaUpdateReader` but forgot?

Reply:
Actually, this name is still suitable even if the return value is changed from `MetaUpdateLogEntry` to `MetaUpdate`, because the iterator returns not only the `MetaUpdate` but also `SequenceNumber` which indicates that this is a log.